### DISCUSSION TITLE
Gluu to Janssen migration

### DISCUSSION THREAD

**Discussion by akhina-gluu** [JanssenProject/jans#5333] [Q&A] [answered]
Hi Team,

We are using gluu 4.4 version . But now we got to know that new supported version is 4.5 or else we need to move to Janssen.
But for this we need to do some testing. Unfortunately while trying with this we are not able to install any Gluu 4.x vesions. Its asking for credentials which was previously not needed. Can someone please help us with this.

Thanks,
Akhina S Kumar

**Comment by nynymike** [comment]
How many applications are there? What protocols do they use?

**Comment by moabu** [comment]
How can we help with your move to Janssen ? Are you using the linux setup or the Cloud native setup ? Either way you should be able to port everything besides passport and SAML which is in the works .

**Reply by akhina-gluu** [reply]
We would like to setup a gluu 4.5 version or any other lower version for testing. But its not allowing us to install or upgrade .Can you help me with this?

**Reply by moabu** [reply, marked as answer]
This is Janssen and isn't the right place for Gluu inquiries .  Please refer to this https://github.com/GluuFederation/cloud-native-edition/issues/558#issuecomment-1598957412

**Reply by akhina-gluu** [reply]
Thanks for the update

**Comment by nynymike** [comment]
Send email to sales@gluu.org if you need access to the Gluu 4 binary repos.
